build

```diff
@@ -4,12 +4,16 @@ cd "$( dirname "${BASH_SOURCE[0]}" )"
 pushd engine > /dev/null
 NODE_ENV=development npm install
 npm run build
+npm pack
 popd > /dev/null
 
 for game in test-games/*; do
     pushd "$game" > /dev/null
     if [ -f package.json ]; then
-        NODE_ENV=development npm install
+        echo installing "$game"
+        NODE_ENV=development npm uninstall engine
+        NODE_ENV=development npm install engine@file:../../engine/jellyfish.js-0.0.2.tgz
```

Comment:
Can you not hardcode the version number? Something like `npm install "engine@file:$(printf '%s\n' ../../engine/jellyfish.js-*.tgz | sort -Vr | head -n1)"` perhaps?